CorsixTH/Lua/app.lua

```diff
@@ -28,7 +28,7 @@ local runDebugger = corsixth.require("run_debugger")
 -- Increment each time a savegame break would occur
 -- and add compatibility code in afterLoad functions
 
-local SAVEGAME_VERSION = 156
+local SAVEGAME_VERSION = 157
```

Comment:
Should be bumped before merged, master is now 157.

Reply:
Done.